glad/__main__.py

```diff
@@ -71,6 +71,11 @@ class GlobalConfig(Config):
         description='Makes the build reproducible by not fetching the latest '
                     'specification from Khronos.'
     )
+    WITH_DOCS = ConfigOption(
```

Comment:
I think I prefer `docs` here.

```suggestion
    DOCS = ConfigOption(
```